From 0c12e77c286d72049be889c25d2f5778aae48bec Mon Sep 17 00:00:00 2001
From: Kovid Goyal <kovid@kovidgoyal.net>
Date: Fri, 21 Aug 2020 09:22:10 +0530
Subject: [PATCH 425/425] Change default for asciiizing filenames on save to
 disk to false

By now most platforms support unicode filenames.
---
 src/calibre/library/save_to_disk.py | 10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/src/calibre/library/save_to_disk.py b/src/calibre/library/save_to_disk.py
index 7b370b67f8..5221d21d63 100644
--- a/src/calibre/library/save_to_disk.py
+++ b/src/calibre/library/save_to_disk.py
@@ -106,12 +106,10 @@ def config(defaults=None):
                 'directory with filenames containing title and author. '
                 'Available controls are: {%(controls)s}')%dict(
                     templ=DEFAULT_SEND_TEMPLATE, controls=', '.join(FORMAT_ARGS)))
-    x('asciiize', default=True,
-            help=_('Normally, calibre will convert all non English characters into English equivalents '
-                'for the file names. '
-                'WARNING: If you turn this off, you may experience errors when '
-                'saving, depending on how well the filesystem you are saving '
-                'to supports unicode.'))
+    x('asciiize', default=False,
+            help=_('Have calibre convert all non English characters into English equivalents '
+                'for the file names. This is useful if saving to a legacy filesystem '
+                'without full support for Unicode filenames.'))
     x('timefmt', default='%b, %Y',
             help=_('The format in which to display dates. %(day)s - day,'
                 ' %(month)s - month, %(mn)s - month number, %(year)s - year. Default is: %(default)s'
-- 
2.28.0

